torchci/lib/bot/retryBot.ts

```diff
@@ -1,10 +1,35 @@
 import { Probot } from "probot";
-import { getFlakyJobsFromPreviousWorkflow } from "../jobUtils";
+import { queryClickhouseSaved } from "../clickhouse";
 import { CachedConfigTracker } from "./utils";
 
 const SUCCESS_CONCLUSIONS = ["success"];
 const FAILURE_CONCLUSIONS = ["failure", "cancelled", "timed_out"];
 
+async function getFlakyJobsFromPreviousWorkflow(
+  owner: string,
+  repo: string,
+  branch: string,
+  workflowName: string,
+  workflowId: number
+): Promise<any> {
+  const flakyJobs = await queryClickhouseSaved("flaky_workflows_jobs", {
+    branches: [branch],
+    maxAttempt: 1, // If the job was retried and still failed, it wasn't flaky
+    nextWorkflowId: `${workflowId}`, // Query the flaky status of jobs from the previous workflow
```

Comment:
should this be a str or an int? does it matter?

Reply:
Good catch, it should be an int.  I mock the query in unit test, so this bug doesn't show up, and there i no good way to test the bot end to end :(